src/services/static-snapshot-service.ts

```diff
@@ -73,19 +74,27 @@ export default class StaticSnapshotService {
   }
 
   _buildLocalUrl() {
-    return `http://localhost:${this.options.port}${this.options.baseUrl}`
+    return `http://localhost:${this.configuration.port}${this.configuration['base-url']}`
   }
 
   async _buildPageUrls() {
-    const baseUrl = this._buildLocalUrl()
-    const pageUrls = [] as any
+    // We very intentially remove '' values from these globs because that matches every file
+    const ignoreGlobs = this.configuration['ignore-files']
+      .split(',')
+      .filter((value) => value !== '')
+
+    const snapshotGlobs = this.configuration['snapshot-files']
+      .split(',')
+      .filter((value) => value !== '')
```

Comment:
previously we were not guarding against `''` values for the snapshot globs, which appear to match everything. This is a bit of a bonus change.